flink-metrics/flink-metrics-prometheus/pom.xml

```diff
@@ -102,34 +102,4 @@ under the License.
 		</dependency>
 	</dependencies>
 
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-shade-plugin</artifactId>
-				<executions>
-					<execution>
-						<id>shade-flink</id>
-						<phase>package</phase>
-						<goals>
-							<goal>shade</goal>
-						</goals>
-						<configuration>
-							<artifactSet>
-								<includes>
-									<include>io.prometheus:*</include>
-								</includes>
-							</artifactSet>
-							<relocations combine.children="append">
```

Comment:
My bad, we should only remove the relocations, but leave the rest. Otherwise we aren't building a fat jar.

Reply:
Sorry,I misunderstood your meaning and have restored the configuration of fatjar